src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -114,44 +114,12 @@ TwitchMessageBuilder::TwitchMessageBuilder(
 
 bool TwitchMessageBuilder::isIgnored() const
 {
-    if (SharedMessageBuilder::isIgnored())
-    {
-        return true;
-    }
-
-    auto app = getApp();
-
-    if (getSettings()->enableTwitchBlockedUsers &&
-        this->tags.contains("user-id"))
-    {
-        auto sourceUserID = this->tags.value("user-id").toString();
-
-        auto blocks =
-            app->accounts->twitch.getCurrent()->accessBlockedUserIds();
-
-        if (auto it = blocks->find(sourceUserID); it != blocks->end())
-        {
-            switch (static_cast<ShowIgnoredUsersMessages>(
-                getSettings()->showBlockedUsersMessages.getValue()))
-            {
-                case ShowIgnoredUsersMessages::IfModerator:
-                    if (this->channel->isMod() ||
-                        this->channel->isBroadcaster())
-                        return false;
-                    break;
-                case ShowIgnoredUsersMessages::IfBroadcaster:
-                    if (this->channel->isBroadcaster())
-                        return false;
-                    break;
-                case ShowIgnoredUsersMessages::Never:
-                    break;
-            }
-
-            return true;
-        }
-    }
-
-    return false;
+    return isIgnoredMessage({
+        /*.message = */ this->originalMessage_,
+        /*.twitchUserID = */ this->tags.value("user-id").toString(),
+        /*.isMod = */ this->channel->isMod(),
+        /*.isBroadcaster = */ this->channel->isBroadcaster(),
```

Comment:
These comments were a part of the old solution, right? Can those be removed now?

Reply:
The comments still serve the purpose to saying which parameter is what, format might be a bit weird since it's part of the old solution but I think it is still pretty enough